tests/beakerlib/variables-support-in-profiles/runtest.sh

```diff
@@ -24,7 +24,7 @@ rlJournalStart
     rlPhaseStartSetup
         rlAssertRpm $PACKAGE
         rlImport "tuned/basic"
-        rlRun "tunedDisableSystemdRateLimitingStart"
+        tunedDisableSystemdRateLimitingStart
```

Comment:
I think all commands (including functions) should be started through `rlRun`, so it's easy to spot possible test problems in a consistent way. But I will let it on @RHEmployee to eventually make such changes. For now, I think it's good to be consistent, thus let's go with this change.